assets/js/towermanager.js

```diff
@@ -0,0 +1,44 @@
+class TowerManager {
+
+	constructor(game) {
+		this.game = game;
+
+		this.towerTypes = [
+		    {
+		        key: 'Cannon'
+		    }
+		];
+
+		this.towerTypes.forEach(tower => {
+			this.game.load.json(tower.key + '_properties', 'assets/towers/' + tower.key + '/properties.json');
+		});
+	}
+
+	load() {
```

Comment:
Formatting is not correct.

Reply:
Thanks, I have to switch sublime to spaces instead of tabs for this project :)